dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/data_config_panel_fields.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import React from 'react';
+import { EuiButtonIcon, EuiPanel, EuiSpacer, EuiText, EuiTitle } from '@elastic/eui';
+import { ConfigListEntry } from '../../../../../../../../common/types/explorer';
+import { VIS_CHART_TYPES } from '../../../../../../../../common/constants/shared';
+import {
+  AGGREGATIONS,
+  CUSTOM_LABEL,
+  GROUPBY,
+} from '../../../../../../../../common/constants/explorer';
+
+export const DataConfigPanelFields = ({
+  list,
+  sectionName,
+  visType,
+  addButtonText,
+  handleServiceAdd,
+  handleServiceRemove,
+  handleServiceEdit,
+}: any) => {
+  const isHeatMapAddButton = (name: string) => {
+    if (visType === VIS_CHART_TYPES.HeatMap) {
+      if (name === AGGREGATIONS) return list?.length === 1;
+      return !(list?.length < 2);
+    }
+    return false;
+  };
+
+  return (
+    <div className="panel_section">
+      <EuiTitle size="xxs" className="panel_title">
+        <h3>{sectionName}</h3>
+      </EuiTitle>
+      <EuiSpacer size="s" />
+      {list !== undefined &&
+        list.map((obj: ConfigListEntry, index: number) => (
+          <>
+            <EuiPanel key={index} paddingSize="s" className="panelItem_button">
+              <EuiText
+                size="s"
+                className="field_text"
+                onClick={() => handleServiceEdit(false, index, sectionName)}
+              >
+                <a role="button" tabIndex={0}>
```

Comment:
Same here and all other places, please use: https://eui.elastic.co/#/navigation/link

Reply:
done